charts/quickwit/values.yaml

```diff
@@ -424,10 +447,16 @@ prometheusRule:
   #      severity: warning
 
 service:
+  # Deprecated, please use <name>.service.type
   type: ClusterIP
 
+  # Deprecated, please use <name>.service.annotations
+  # or service.headless.annotations for the headless service
   annotations: {}
 
+  headless:
+    annotations: {}
```

Comment:
It feels a bit weird to have this config for the headless here. I think we can keep the current behavior where the headless service only wears the common service annotations. 

If we go with this, and the 2 comments above as well as https://github.com/quickwit-oss/helm-charts/pull/84, the service section would become:
```
service:

  # annotations common to all Quickwit services
  annotations: {}

  # -- Set the ip family policy to configure dual-stack. See [Configure dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services)
  # ipFamilyPolicy: "PreferDualStack"

  # -- Sets the families that should be supported and the order in which they should be applied to ClusterIP as well. Can be IPv4 and/or IPv6.
  # ipFamilies: ["IPv4","IPv6"]
```

Reply:
works for me!

Reply:
as discussed here, this will now look like:

```
service:
  # Service type configuration default for all Quickwit services
  type: ClusterIP

  # annotations common to all Quickwit services
  annotations: {}

  # -- Set the ip family policy to configure dual-stack. See [Configure dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services)
  # ipFamilyPolicy: "PreferDualStack"
  # -- Sets the families that should be supported and the order in which they should be applied to ClusterIP as well. Can be IPv4 and/or IPv6.
  # ipFamilies: ["IPv4","IPv6"]
```